wordless/theme_builder/vanilla_theme/webpack.config.js

```diff
@@ -61,7 +64,11 @@ module.exports = (env) => {
           test: /\.(jpe?g|png|gif|svg)$/i,
           loader: 'file-loader',
           options: {
-            name: '[path][name].[ext]',
+            name (resourcePath) {
+              resourcePath = resourcePath.replace(path.join(srcDir, `/${imageFolder}/`), '');
+              return resourcePath;
+            },
+            outputPath: imageFolder
```

Comment:
```suggestion
            name: '[name].[ext]',
            context: path.join(srcDir),
            outputPath: (_url, resourcePath, context) => {
              return resourcePath.replace(context, '')
            }
```

this suggested code is just equivalent to the previous one, but the function to manage the custom logic is moved to the `outputPath` key, which is more striclty responsible to choose the path; while the `name` key is really and only just responsible of the file naming.
I've also used a custom `context` just in order to diminsh the complexity inside the function.

It's really just a matter of readibility, not functionality. So feel free to choose your favourite approach.